generator/generator.go

```diff
@@ -430,6 +434,10 @@ func main() {
 		ListBuilderKeyThreshold: *listBuilderKeyThreshold,
 		GenerateWildcardPaths:   *generateWildcardPaths,
 		SimplifyWildcardPaths:   *simplifyWildcardPaths,
+		TrimOCPackage:           *trimOCPackage,
+		FakeRootPackageName:     *fakeRootPackageName,
+		SplitByModule:           *splitByModule,
+		BaseImportPath:          *baseImportPath,
```

Comment:
Maybe add a validation here that if `SplitByModule=true`, then the fake root name and the import path must not be empty.

Reply:
added check for import path. fake root has a default in ypathgen